windows-nsis/openvpn.nsi

```diff
@@ -244,7 +250,11 @@ Section /o "${PACKAGE_NAME} Service" SecService
 
 	SetOutPath "$INSTDIR\bin"
 	File /oname=openvpnserv2.exe "${OPENVPNSERV2_EXECUTABLE}"
-	File "${OPENVPN_ROOT}\bin\openvpnserv.exe"
+        ${If} ${RunningX64}
+                File "${OPENVPN_ROOT_X86_64}\bin\openvpnserv.exe"
+        ${Else}
+                File "${OPENVPN_ROOT_I686}\bin\openvpnserv.exe"
+        ${EndIf}
```

Comment:
@chipitsine : I always use "git diff" to have a look at the changes before commit: in general it reveals these tabs vs. spaces quite nicely.